### ISSUE TITLE
Web pages do not display information

### ISSUE THREAD

**Issue by yvyang** [emqx/emqx#2534]
#### Environment
Raspberry Pi 3B
- OS: Linux raspberrypi 4.19.40-v7
- Erlang/OTP: 
- EMQ: EMQ X Cube

#### Description
Web pages(https://cube.emqx.io/console) do not display information.
Online, but does not display relevant information.
Only the storm.username and storm.password have been changed.

**Comment by Gilbert-Wong** [comment]
Hi, @yvyang, the `cube.emqx.io` does not support to modify username and password. And the config entry `storm.username` and `storm.password` has been changed to `cube.username` and `cube.password` in emqx edge 3.2 release.

**Comment by yvyang** [comment]
The latest version is V3.1.1.when will V3.2 be released?
If change ```storm``` to ```cube```,Prompt error message.

**Comment by Gilbert-Wong** [comment]
Even if replace `storm` with `cube`, there should be no error. And could you please give us more details about the problem you encountered. The cube.emqx.io should not be able to let the users modify username and password. Why did you have to change the `storm.username` and `storm.password`?

**Comment by yvyang** [comment]
Thank you for your reply.My English is not good. Maybe I have a problem with my expression.
---
1. in the help document(Chinese)【https://cube.emqx.io/docs/zh-hans/quick_start/connect_cube.html】
the config file is ```etc/plugins/emqx_storm.conf```
![QQ截图20190513162706](https://user-images.githubusercontent.com/22268943/57607144-fc509c00-759c-11e9-9078-505f1725fe77.png)
2. in the help documen(English)【https://cube.emqx.io/docs/en/quick_start/connect_cube.html】
the config file is ```etc/plugins/emqx_cube.conf```
![QQ截图20190513163442](https://user-images.githubusercontent.com/22268943/57607217-273af000-759d-11e9-8d31-b6b3f906fa97.png)
3. There are don't have ```emqx_cube.conf``` in the folder,I can only change the ```emqx_storm.conf```.
4. But,The content of this file ```etc/plugins/emqx_storm.conf``` is:
**Unlike Help Documents**
```
storm.address = storm.emqx.io:1883
storm.username = qg3rgewt135
storm.password = e3t5tt34tw3t
```
5. I changed it to the following
```
storm.address = storm.emqx.io:1883
storm.username = daf78690-72db-11e9-bbe2-4d5b798ec2e0
storm.password = 0bcd2c64-a6ea-417b-8208-20ed49a16de9
```
6. Web pages do not display relevant information after startup
**Help Documents**
![image](https://user-images.githubusercontent.com/22268943/57607620-0d4ddd00-759e-11e9-9dc2-9d6a1e51131f.png)
**My web page**
![page](https://user-images.githubusercontent.com/22268943/57606017-8e0ada00-759a-11e9-8c5f-6cf0771a540c.png)

**Comment by wivwiv** [comment]
@yvyang 

#### We renamed the product name from storm to cube
 - Config file: `etc/plugins/emqx_storm.conf ` --> `etc/plugins/emqx_cube.conf `
- Access point address: `storm.*` -> `cube.*`
- The Cube platform only working on EMQ X [emqx-cube](http://github.com/emqx/emqx-cube) plugins

We made a mistake that we did not update the plug-in in v3.1.1. At present, the plug-in can only be used online. We will fix this problem in v3.1.2. @Gilbert-Wong @turtleDeng 

Now you can also build and replace emqx-cube by yourself.
